tools/api-markdown-documenter/src/documentation-domain/DocumentNode.ts

```diff
@@ -102,7 +76,7 @@ export class DocumentNode implements UnistParent<SectionNode>, DocumentNodePrope
 	public readonly frontMatter?: string;
 
 	public constructor(properties: DocumentNodeProperties) {
-		this.documentItemMetadata = properties.documentItemMetadata;
+		this.apiItem = properties?.apiItem;
```

Comment:
You shouldn't need the `?` here.
```suggestion
		this.apiItem = properties.apiItem;
```